src/api/cubrid_log.c

```diff
@@ -631,6 +677,10 @@ cubrid_log_connect_server_internal (char *host, int port, char *dbname)
       g_conn_entry = NULL;
     }
 
+#if defined (WINDOWS)
+  (void) css_windows_shutdown ();
```

Comment:
In no `cubrid_log_error` case (when successful), it also seems to have to be done.

Reply:
`css_windows_startup` should be maintained until the CDC API is terminated (cubrid_log_finalize()). 
Because, after connected to the server, it communicates with the server through the API ( get lsa from the server , get log items from the server, ...).

Reply:
Oh, I see. I didn't know the lifecycle. Then, another question: if `css_windows_startup()` failes, `cubrid_log_finalize()` is never called?

Reply:
yes, if css_windows_startup() fails, cubrid_log_connect_server () will return error to the client (error handling will be added for `css_windows_startup()` according to @hornetmj 's review) 

If user gets an error from `cubrid_log_connect_server()`, user have to try again with `cubrid_log_connect_server()`. 
Nothing can be called without `cubrid_log_connect_server()`'s success

Reply:
Thank you for the elaboration.